util_test.go

```diff
@@ -74,17 +64,14 @@ func testParseTime(t *testing.T, input interface{}, expected time.Time) {
 }
 
 func TestParseFloat64(t *testing.T) {
-	testParseFloat64(t, "123", 123.0)
```

Comment:
I think it's safer to only allow json numbers as numbers, and strings should be compared as strings, but am I missing something?